docs/core/diagnostics/dumps.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Dumps - .NET
+description: An introduction to dumps in .NET.
+ms.date: 10/12/2020
+---
+
+# Dumps
+
+A dump is a file that contains snapshot of the process at the time they were created and can be useful for examining the state of your application for debugging. Dumps can be used for post-mortem debugging of your .NET application when it is difficult to attach a debugger to it (for example, when the app is running in production). Using dumps allows you to capture the state of the problematic process and examine it without having to stop it from running while being analyzed.
+
+## Collect dumps
+
+Dumps can be collected in a variety of ways depending on which platform you are running your app on.
+
+> [!NOTE]
+> Collecting a dump inside a container requires PTRACE capability which can be added via `--cap-add=SYS_PTRACE` or `--privileged`.
```

Comment:
Another note would be: know that these contain all the information in the process - handle with care as it could contain sensitive information.